lib/ansible/plugins/action/__init__.py

```diff
@@ -346,8 +346,11 @@ def _execute_module(self, module_name=None, module_args=None, tmp=None, task_var
         # set no log in the module arguments, if required
         if self._play_context.no_log:
             module_args['_ansible_no_log'] = True
+            module_display_args = "(no_log enabled, args censored)"
+        else:
+            module_display_args = module_args
 
-        self._display.vvv("Module: %s, args: %s" % (module_name, module_args))
+        self._display.vvv("Module: %s, args: %s" % (module_name, module_display_args))
```

Comment:
i removed this as even if you respect the play's no_log there is no current way to respect the module spec's no_log